core-samples/jsapi-custom-ui/package.json

```diff
@@ -8,12 +8,12 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@arcgis/core": "~4.32.0",
-    "react": "^19.0.0",
-    "react-dom": "^19.0.0"
+    "@arcgis/core": "~4.33.0",
+    "react": "^19.1.0",
+    "react-dom": "^19.1.0"
   },
   "devDependencies": {
-    "@vitejs/plugin-react": "^4.3.4",
-    "vite": "^6.1.1"
+    "@vitejs/plugin-react": "^4.5.2",
+    "vite": "^6.3.5"
```

Comment:
Update to Vite 7? https://vite.dev/blog/announcing-vite7.html

Reply:
I'm thinking we might want to update to Vite 7 in a  follow-up PR so we can do some more testing first.